doc/documentation.rst

```diff
@@ -0,0 +1,681 @@
+:orphan:
+
+.. _documentation:
+
+Documentation
+=============
+
+.. raw:: html
+
+    <style class='text/css'>
+    .panel-title a {
+        display: block;
+        padding: 5px;
+        text-decoration: none;
+    }
+
+    .plus {
+        float: right;
+        color: #212121;
+    }
+
+    .panel {
+        margin-bottom: 3px;
+    }
+
+    .example_details {
+        padding-left: 20px;
+        margin-bottom: 10px;
+    }
+
+    </style>
+
+This is where you can learn about all the things you can do with MNE. It contains **Background information** and **Tutorials** for taking a deep-dive into the techniques that MNE-python covers. You'll find practical information on how to use these methods with your data, and in many cases some high-level concepts underlying these methods.
```

Comment:
I think we should actually un-capitalize these as they come across as links or specific sections, when really they're just descriptions of types of information. WDYT?